### ISSUE TITLE
Chef issue

### ISSUE THREAD

**Issue by aitala** [btopro/nittany-vagrant#1]
Apparently something changed in Chef 12.0 ...

==> default: [2015-02-10T21:25:58+00:00] ERROR: Cookbook loaded at path(s) [/tmp/vagrant-chef/6549f2b6ccce742569fa5535611c9642/cookbooks/apache2] has invalid metadata: The `name' attribute is required in cookbook metadata
==> default: [2015-02-10T21:25:58+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)

See: https://github.com/chef/chef/issues/2435

**Comment by btopro** [comment]
yeah, I did a ton of work to get elmsln-vagrant up and going again. this is not a simple fix most likely
